src/activate.ts

```diff
@@ -0,0 +1,64 @@
+import * as os from 'os'
+import * as osPath from 'path'
+import * as core from '@actions/core'
+import { executeGetOutput, pixiCmd } from './util'
+
+type ShellHook = {
+  environment_variables: Record<string, string>
+}
+
+const splitEnvironment = (shellHook: ShellHook): [Record<string, string>, string?] => {
+  if (os.platform() === 'win32') {
+    // On Windows, environment variables are case-insensitive but JSON isn't...
+    const pathEnvs = Object.keys(shellHook.environment_variables).filter((k) => k.toUpperCase() === 'PATH')
+    const caseSensitivePathName = pathEnvs[0]
+    if (pathEnvs.length > 0) {
```

Comment:
```suggestion
    if (pathEnvs) {
```
Isn't this equivalent?

Reply:
Possibly, I always prefer to be explicit :eyes:

Reply:
Ok, I checked, it's not the same